.github/workflows/build.yml

```diff
@@ -123,6 +123,7 @@ jobs:
     steps:
       - run: |
           echo "OS_ARCH=`uname -m`" >> $GITHUB_ENV
+          sudo apt -y install lz4
```

Comment:
Rather than `lz4`, I think you can get away with `liblz4-tool`. Along those lines, I think you should just go ahead and add all the packages we detail in our [BUILDING.md](https://github.com/bottlerocket-os/bottlerocket/blob/develop/BUILDING.md). If some of those packages aren't necessary then they should be removed from that doc.